src/web/components/SharingPermission/BulkAddPermissions.tsx

```diff
@@ -203,7 +200,7 @@ export function BulkAddPermissions({
             {recommendationContent}
           </Collapsible>
         )}
-        {hasSharingParticipants && (
+        {(hasSharedSiteIds || sharedTypes.length > 0) && (
```

Comment:
I think the publisher will by default share with DSPs? and I remember we want to add a flag to check if the user has gone through the recommendation? Are you going to do it in a separate PR?

Reply:
Yeah those are separate tickets, see: https://atlassian.thetradedesk.com/jira/browse/UID2-1735 and https://atlassian.thetradedesk.com/jira/browse/UID2-1736